examples/features/functions.effekt.md

```diff
@@ -0,0 +1,51 @@
+# Functions
+
+For defining a function, you may use similar syntax for know from other languages such as Scala:
```

Comment:
Nitpick: "...similar syntax for know from other languages..." doesn't make sense. Maybe remove "for know"?

Reply:
(also thanks for this, currently learning some Effekt and this is useful)